src/test/java/pages/HomePage.java

```diff
@@ -3,18 +3,27 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import org.testng.Assert;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
 
-public class HomePage extends BasePage{
+import java.time.Duration;
+
+public class HomePage extends BasePage {
     public HomePage(WebDriver givenDriver) {
         super(givenDriver);
     }
 
     By avatar = By.cssSelector(".avatar");
+    public WebElement getAvatar(){
+        return waitUntilVisible(avatar);
+    }
 
-
-    public boolean getAvatar(){
-        return driver.findElement(avatar).isDisplayed();
+    public WebElement waitUntilClickable (By element){
+        return new WebDriverWait(driver, Duration.ofSeconds(4)).until(ExpectedConditions.elementToBeClickable(element));
     }
 
+
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```